server/utils/utils.ts

```diff
@@ -21,3 +24,46 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
+export const parseDate = (date: string, fromFormat = 'yyyy-MM-dd') => {
+  if (!date) return null
+  return parse(date, fromFormat, new Date())
+}
+
+export const formatDate = (date: string | Date, fmt = 'd MMMM yyyy') => {
+  if (!date) return undefined
+  const richDate = typeof date === 'string' ? parseISO(date) : date
+  if (!isValid(richDate)) return undefined
+  return format(richDate, fmt)
+}
+
+export const parseDatePickerDate = (datePickerDate: string): Date => {
+  if (!datePickerDate) return null
+
+  const dateFormatPattern = /(\d{1,2})([-/,. ])(\d{1,2})[-/,. ](\d{2,4})/
+
+  if (!dateFormatPattern.test(datePickerDate)) return new Date(NaN)
+
+  const dateMatches = datePickerDate.match(dateFormatPattern)
+
+  const separator = dateMatches[2]
+  const year = dateMatches[4]
+
+  const date = parse(datePickerDate, `dd${separator}MM${separator}${'y'.repeat(year.length)}`, startOfToday())
+  if (!isValid(date)) return new Date(NaN)
+
+  return date
+}
+
+export const simpleDateToDate = (date: { day: string; month: string; year: string }): Date =>
```

Comment:
Some (most?) of these seem to be unused. I guess copied from BVLS? I would prefer we only import stuff we're actually using but if you need them imminently, it's fine to leave them as long we import the tests as well.